code/modules/jobs/job_types/roguetown/peasants/nightman.dm

```diff
@@ -3,13 +3,13 @@
 	flag = NIGHTMASTER
 	department_flag = PEASANTS
 	faction = "Station"
-	total_positions = 0 
-	spawn_positions = 0 // Disabled until he gets his own place on the map
+	total_positions = 1 
+	spawn_positions = 1 // Disabled until he gets his own place on the map
```

Comment:
I take it that comment is no longer necessary since he's getting his own place and may lead to confusion?